tests/subsys/zigbee/zboss_api/callback_api/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8)
+
+add_definitions(-Wno-packed-bitfield-compat)
```

Comment:
Why is this needed here.
In Zephyr, those warnings are handled in the toolchain setup:
https://github.com/NordicPlayground/fw-nrfconnect-zephyr/blob/master/cmake/compiler/gcc/target_warnings.cmake
and can be controlled using `-DW=[1|2|3]`.

Reply:
This warning is generated by ZBOSS headers. The same definition is inside ZBOSS's CMakeLists.txt file. I'd remove it here and move the discussion to the nrfxlib's PR.

Reply:
Line removed. 
This warning suppression is not specified in a sample or test but propagated from ZBOSS CMakeLists.txt